deploy/olm-catalog/open-liberty-operator-certified/open-liberty-operator.v0.5.0.clusterserviceversion.yaml

```diff
@@ -1,16 +1,16 @@
 apiVersion: operators.coreos.com/v1alpha1
```

Comment:
This should be in a new file `open-liberty-operator.v0.5.1.clusterserviceversion.yaml` since 0.5.0 for OL operator was already published.